receiptui/src/main/java/com/hyperwallet/android/ui/receipt/viewmodel/TabbedListReceiptsViewModel.java

```diff
@@ -0,0 +1,145 @@
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package com.hyperwallet.android.ui.receipt.viewmodel;
+
+import android.os.Handler;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.hyperwallet.android.Configuration;
+import com.hyperwallet.android.Hyperwallet;
+import com.hyperwallet.android.exception.HyperwalletException;
+import com.hyperwallet.android.listener.HyperwalletListener;
+import com.hyperwallet.android.model.Errors;
+import com.hyperwallet.android.model.transfermethod.PrepaidCard;
+import com.hyperwallet.android.model.user.User;
+import com.hyperwallet.android.ui.common.repository.Event;
+import com.hyperwallet.android.ui.transfermethod.repository.PrepaidCardRepository;
+import com.hyperwallet.android.ui.user.repository.UserRepository;
+
+import java.util.List;
+
+public class TabbedListReceiptsViewModel extends ViewModel {
+    private UserRepository mUserRepository;
+    private PrepaidCardRepository mPrepaidCardRepository;
+    public MutableLiveData<User> mUser = new MutableLiveData<User>();
+    public MutableLiveData<List<PrepaidCard>> mPrepaidCards = new MutableLiveData<List<PrepaidCard>>();
+    public MutableLiveData<Event<Errors>> mErrors = new MutableLiveData<Event<Errors>>();
+    public ProgramModel mProgramModel;
+    private boolean mIsInitialized;
+
+    public TabbedListReceiptsViewModel(
+            UserRepository mUserRepository,
+            PrepaidCardRepository prepaidCardRepository) {
+        this.mUserRepository = mUserRepository;
+        this.mPrepaidCardRepository = prepaidCardRepository;
+    }
+
+    public void initialize() {
+        if (!mIsInitialized) {
+            mIsInitialized = true;
+            getProgramModel();
+            loadUser();
+        }
+    }
+
+    private void loadUser() {
+        mUserRepository.loadUser(new UserRepository.LoadUserCallback() {
+            @Override
+            public void onUserLoaded(@NonNull User user) {
+                TabbedListReceiptsViewModel.this.mUser.postValue(user);
+                loadPrepaidCards();
+            }
+
+            @Override
+            public void onError(@NonNull Errors errors) {
+                TabbedListReceiptsViewModel.this.mErrors.postValue(new Event(errors));
+            }
+        });
+    }
+
+    private ProgramModel getProgramModel() {
+        Hyperwallet.getDefault().getConfiguration(new HyperwalletListener<Configuration>() {
+            @Override
+            public void onSuccess(@Nullable Configuration result) {
+                if (result != null) {
+                    mProgramModel = ProgramModel.valueOf(result.getProgramModel());
```

Comment:
check if result.getProgranModel() is not as empty string, if its empty its caues crash.